### PR TITLE
Add color to Card and little refactoring

### PR DESCRIPTION
**Pull request by ArturWincenciak**
related issue #10

### CODE REVIEW

File: src/Blef.GameLogic/Suit.cs [hunk 1]
```diff
@@ -0,0 +1,10 @@
+namespace Blef.GameLogic
+{
+    public enum Suit
+    {
+        Diamonds, //dzwonek, karo
```
Comment (kmorcinek) [inline src/Blef.GameLogic/Suit.cs:5]: Nice comments, kocham

File: src/Blef.GameLogic/PokerHands/PokerHand.cs [hunk 2]
```diff
@@ -1,29 +1,36 @@
...
 {
-    public abstract class PokerHand
+    public abstract class PokerHand : IComparable<PokerHand>
     {
         public abstract bool IsOnTable(IEnumerable<Card> cards);
 
         /// <summary>
-        /// Ranks should be assigned according to <see href="https://pl.wikipedia.org/wiki/Blef_(gra)"/> 
+        /// Ranks should be assigned according to 
+        /// <see href="https://pl.wikipedia.org/wiki/Blef_(gra)"/>
```
Comment (kmorcinek) [inline src/Blef.GameLogic/PokerHands/PokerHand.cs:12]: Look nicer

File: src/Blef.GameLogic/PokerHands/HighCard.cs [hunk 3]
```diff
@@ -6,33 +6,26 @@ namespace Blef.GameLogic.PokerHands
...
         {
-            HighCard highCard = value as HighCard;
-
-            if (highCard == null)
-            {
-                throw new InvalidOperationException($"Can compare only {nameof(PokerHand)} with the same rank");
-            }
-
-            return Card - highCard.Card;
+            return (int)Card.Rank;
```
Comment (kmorcinek) [inline src/Blef.GameLogic/PokerHands/HighCard.cs:28]: Now I see that my implementation and idea were completely not clear. This whole will be changed to something else

File: src/Blef.GameLogic/Card.cs [hunk 4]
```diff
@@ -1,12 +1,72 @@
-﻿namespace Blef.GameLogic
+using System;
+using System.Collections.Generic;
+
+namespace Blef.GameLogic
 {
-    public enum Card
+    public class Card : IComparable<Card>
```
Comment (kmorcinek) [inline src/Blef.GameLogic/Card.cs:6]: I see no point in comparing separate cards, what we really will compare are Poker hands.

**Review by kmorcinek — COMMENTED** [review]
Ale fajnie, moja piersze review na Github :)

File: src/Blef.GameLogic/Card.cs (hunk 4 above)
Comment (ArturWincenciak) [reply src/Blef.GameLogic/Card.cs:6]: You are right. Yesterday at 2 am I sow point but now I don't see too.

**Comment by ArturWincenciak** [comment]
Is too late to merge this pull request. I'm closing it.

**Pull request closed without merging**